.travis.yml

```diff
@@ -14,7 +14,9 @@ cache:
 
 env:
   matrix:
-    - SYMFONY_VERSION=2.8.* 
+    - SYMFONY_VERSION=2.8.*
+    - SYMFONY_VERSION=3.1.*
```

Comment:
please don't do this but instead add to the matrix.include below. if you add symfony versions up here, the tests are ran for each php version X every symfony version, which leads to a ton of unnecessary builds. i guess we should build the latest symfony (aka 3.2.*) with every php version, and add the others below to be built once